### DISCUSSION TITLE
Anyone have a high memory usage by nvim extension ?

### DISCUSSION THREAD

**Discussion by RigottiG** [vscode-neovim/vscode-neovim#1664] [Q&A]
Has anyone experienced high memory usage with the NVim extension for Visual Studio Code? I've noticed that my extension is consuming a significant amount of memory, and I'm wondering if this is a common issue among other users. If you've encountered this problem, how did you manage to resolve or mitigate it? Any insights or suggestions would be greatly appreciated.

![image](https://github.com/vscode-neovim/vscode-neovim/assets/13246770/f0b72418-2621-47f1-b2e5-7591196a7f12)

**Comment by justinmk** [comment]
Depends on the Nvim plugins you have installed?

**Reply by RigottiG** [reply]
I'm using a fresh install of LazyVim only for test purposes

**Reply by justinmk** [reply]
LazyVim is gigantic

**Reply by RigottiG** [reply]
![image](https://github.com/vscode-neovim/vscode-neovim/assets/13246770/255b061a-f907-440a-91c1-e4d25656f345)

I remove my nvim config and the consume is the same :(
